c31-swynlake/reactive-nutshell
# Bm friends link

# Description

Adding link to each username in chat room and function to add them as a friend.

Applies to issue # (8)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout bm-friends-link`
3. go to chatroom. verify that if you click on someones name you are already friends with it does not add them and that you can add someone you are not friends with"


# Checklist:
